### PR TITLE
Changes to fix Polish and Greek subtitle encoding

### PR DESCRIPTION
**Pull request by aggelos81**
These changes can be applied to any subtitle affected but we need to have the encoding that the subtitles should be read with!

### CODE REVIEW

File: PopcornTime/VLC/SYVLCPlayerViewController.m [hunk 1]
```diff
@@ -1311,15 +1311,30 @@ - (void) newSubSelected
...
+                string = (__bridge NSString *)cfstring;
+                CFRelease(cfstring);
+            }
+            if([name isEqualToString:@"Polish"]){
+                NSData* dataFile = [NSData dataWithContentsOfFile:path];
+                CFStringRef cfstring = CFStringCreateWithBytes(kCFAllocatorDefault, dataFile.bytes, dataFile.length, kCFStringEncodingISOLatin2, YES);
+                string = (__bridge NSString *)cfstring;
+                CFRelease(cfstring);
+            }
     }
```
Comment (mtwww) [inline PopcornTime/VLC/SYVLCPlayerViewController.m]: if([name isEqualToString:@"Turkish"]){
            NSData* dataFile = [NSData dataWithContentsOfFile:path];
            CFStringRef cfstring = CFStringCreateWithBytes(kCFAllocatorDefault, dataFile.bytes, dataFile.length, kCFStringEncodingISOLatin5, YES);
            string = (__bridge NSString *)cfstring;
            CFRelease(cfstring);
        }

**Comment by buzzel** [comment]
We are working on a rewrite of subtitles so we are holding off from merging your PR in. The plan is to switch to OpenSubs, but their API is confusing to work with.

**Comment by aggelos81** [comment]
Well if I can help with anything I would be glad to do so.

**Comment by buzzel** [comment]
I merged in some fixes so feel free to have a look and see how it works. Feel free to tackle any bug fixes or tweaks you would like to see done. If its worth adding in, we'll merge it in :)

**Pull request closed without merging**